rules/yoghurts.txt

```diff
@@ -0,0 +1,27 @@
+# Product rules to categorize yoghurts more finely based on their ingredients
+
+# General rules syntax:
+
+# name: a name to identify the rule
+
+# description: an explanation of what the rule does
+
+# conditions: 1 or more conditions, in the same format as the search API v2 (makes copy and pasting easy to test rules)
+# e.g. https://world.openfoodfacts.org/search?categories_tags_en=yoghurts,-sweetened%20yoghurts&ingredients_tags_en=sugar
+# - conditions are comma separated, and can also be specified on several conditions: lines
```

Comment:
you should detail:
do we do an "and" on all conditions ?
Or do we do an "and" on conditions on same line, and "or" between lines ?
Or do we do an "and" on conditions expressed with "&" sign, and an or between those, separated by comma or line ?

Reply:
We do an "and" on all conditions. Initially I thought to have only 1 conditions: line, but then I'm thinking it could be more readable to have multiple lines.

e.g. in yoghurts.txt we could have all rules targeting yoghurts, and so the first conditions: line could be just to check that the category is yoghurts.